finance-api/api-reference.md

```diff
@@ -0,0 +1,288 @@
+|accountTypeGroup |string |path |required |The type of a merchant's Zettle account. You can use one of the following account types: <br/><ul><li> `PRELIMINARY` account where transactions are to be confirmed by third-party acquirers.</li><li> `LIQUID` account where transactions are to be paid out to the merchant.</li></ul>
+|start |string |query |required |A start time in UTC (inclusive) from when the transactions will be fetched. You can specify a UTC time in one of the following formats: <br/><ul><li>`YYYY-MM-DD` to specify a date. For example, `2020-11-29`.</li><li>`YYYY-MM-DDThh:mm:ss` to specify a time. For example, `2020-11-29T03:10:02`.</li></ul>
+|end |string |query |required |An end time in UTC (exclusive) before when the transactions will be fetched. You can specify a UTC time in one of the following formats: <br/><ul><li>`YYYY-MM-DD` to specify a date. For example, `2020-11-29`.</li><li>`YYYY-MM-DDThh:mm:ss` to specify a time. For example, `2020-11-29T03:10:02`.</li></ul>
+|includeTransactionType |string |query |optional |Which transaction types to fetch. <br>You can include more than one [supported transaction types](#supported-transaction-types) in a request.  
+|limit |integer |query |optional |The maximum number of transactions to return in a response. You can specify `limit` with any integer greater than 0. Use `limit` and `offset` together to set response pagination.<br/>For example, to return three transaction at a time for all transactions during a specific period, set `limit` as `3` and `offset` as `0` in the first request. Then set `limit` as `3` and `offset` as `0` in the second request. 
+|offset |integer |query |optional |The number of transactions to skip before beginning to return in a response. You can specify `offset` with any integer greater than or equal to 0.  Use `limit` and `offset` together to set response pagination.
+</details>
+
+#### Supported transaction types
+> **Note:** Deprecated transaction types are no longer in use, but may appear in historic data.
+<details open="true">
+<summary>Supported transaction types</summary>
+
+|Type |Description
+|:---- |:----
+|ADJUSTMENT |A bookkeeping adjustment.
+|ADVANCE |The cash advance given by Zettle to a merchant. A cash advance is a type of financing that is offered to merchants based on their sales history. The advance is paid back with monthly down payments.
+|ADVANCE_DOWNPAYMENT |A down payment on a previously paid out cash advance.
+|ADVANCE_FEE_DOWNPAYMENT |The netting of a cash advance fee.
+|BANK_ACCOUNT_VERIFICATION (Deprecated) |No available description.
+|CARD_PAYMENT |A card payment. Contains a reference to the card payment in the Purchase API.
```

Comment:
Not only CARD_PAYMENT transactions contain the reference to the payment that can be found using Purchase API.